crates/pixi_manifest/src/manifest.rs

```diff
@@ -463,22 +471,35 @@ impl Manifest {
         Ok(())
     }
 
+    /// Handles the target missing error cases
     fn handle_target_missing(
         platform: Option<&Platform>,
         feature_name: &FeatureName,
+        section: &str,
```

Comment:
wdyt about making section an enum? I think we have a finite list of supported sections, so maybe having a more strict variant will help with development?

Reply:
I changed it to use constants instead, added `dependencies` constant should add that to more places in a later PR.